ebml/EbmlElement.h

```diff
@@ -324,7 +324,7 @@ class EBML_DLL_API EbmlSemantic {
     const EbmlCallbacks & GetCallbacks;
 };
 
-typedef const class EbmlSemanticContext & (*_GetSemanticContext)();
+using _GetSemanticContext = const class EbmlSemanticContext &(*)();
```

Comment:
This kind of change seems to be useless in this context https://stackoverflow.com/questions/10747810/what-is-the-difference-between-typedef-and-using-in-c11

Reply:
It's equivalent, yes.

Reply:
It is, but I vastly prefer `using` as it can be used in more contexts (e.g. template functions). The [C++ core guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rt-using) recommend using `using` over `typedef` as well; reasoning:

> Improved readability: With using, the new name comes first rather than being embedded somewhere in a declaration. Generality: using can be used for template aliases, whereas typedefs can’t easily be templates. Uniformity: using is syntactically similar to auto.

So this is a :+1: from me, too.